code/modules/reagents/reagent_containers.dm

```diff
@@ -12,6 +12,7 @@
 	var/spawned_disease = null
 	var/disease_amount = 20
 	var/spillable = FALSE
+	var/you_drink_from_this = FALSE

```

Comment:
Bad variable name

Reply:
blame flattest for the original pr, ill change it.

Reply:
done.